docs/source/commands/ledgerutil.md

```diff
@@ -0,0 +1,64 @@
+<!---
+ File generated by help_docs.sh. DO NOT EDIT.
+ Please make changes to preamble and postscript wrappers as appropriate.
+ --->
+
+# ledgerutil
+
+The `ledgerutil compare` command allows administrators to compare channel snapshots from two different peers.
+Although channel snapshots from the same height should be identical across peers, if the snapshot
+files indicate different state hashes (as seen in `_snapshot_signable_metadata.json` file from each snapshot)
+then it indicates that at least one of the peers state databases is not correct relative to the blockchain.
+The `ledgerutil compare` utility will output a set of JSON files if the snapshots are not identical.
+One output JSON file will include any key/value differences sorted by key, another JSON file will include
+any key/value differences sorted by block and transaction height so that you can identify the height
+where a divergence may have occurred.
+
```

Comment:
I guess adding about potential use of the utility may be helpful  -- e..g., in the case of difference between two snapshots, users may like to figure out which snapshot is diverged from the expected, perhaps either based on application knowledge or comparing with more snapshots from other peers. The logs near the time of initial divergence may help figure out what may have gone wrong.